security-policies/bundle/compliance/cis_azure/rules/cis_4_2_3/rule.rego

```diff
@@ -0,0 +1,29 @@
+package compliance.cis_azure.rules.cis_4_2_3
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import data.compliance.policy.azure.sql_server.ensure_vulnerability_assessment_storage_account as audit
+import future.keywords.every
+import future.keywords.if
+
+finding = result if {
+	# filter
+	data_adapter.is_sql_server
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(va_recurrent_scans_enabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default va_recurrent_scans_enabled = false
+
+va_recurrent_scans_enabled if {
+	count(data_adapter.resource.extension.sqlVulnerabilityAssessmentSettings) > 0
+
+	every setting in data_adapter.resource.extension.sqlVulnerabilityAssessmentSettings {
+		audit.ensure_vulnerability_assessment_storage_account(setting)
+		setting.recurringScansEnabled == true
```

Comment:
Just to clarify, this part check that:

> Ensure that value for parameter `RecurringScansInterval` is not set to `None`.

Reply:
Yes. The `None` is a logic performed in the powershell implementation (https://github.com/Azure/azure-powershell/blob/main/src/Sql/Sql/VulnerabilityAssessment/Services/BaseSqlVulnerabilityAssessmentAdapter.cs#L114)

I decided to skip the powershell conversion and only check the boolean